site/docs/ch/meta-service/table-management.md

```diff
@@ -0,0 +1,57 @@
+# 表管理
+[AMS Dashboard](http://localhost:1630)中提供了命令行工具`Terminal`帮组用户快速完成表的创建、修改与删除操作。
+同时还可以在[Spark](#flink.md)和[Flink](#spark/spark-ddl.md)等引擎中使用SQL完成表的管理。
```

Comment:
The links between flink and spark are out of order.

Reply:
I will fix it